Unix/tests/cli/test_cli.cpp

```diff
@@ -151,6 +151,32 @@ static int StartServer()
     travisCI = true;
 #endif
 
+    //
+    // kill running omiserver if have any
+    //
+
+    
+    {
+        char buf[64];
+
+        int pid = -1;
+        int status = PIDFile_Read(&pid);
+
+        if (pid != -1)
+        {
+
+#if defined(aix)
+            sprintf(buf, "sudo kill -TERM %d", pid);
+#elif !defined(solaris) && !defined(linux)
+            sprintf(buf, "killall -TERM %d", pid);
```

Comment:
If you have a specific PID, what's the benefit of `killall` or `pkill`, which generally kill groups of processes by name? With a specific PID, why not just use `kill` universally?